config/zones/XK.yaml

```diff
@@ -4,10 +4,18 @@ bounding_box:
   - - 21.791473
     - 43.269883
 capacity:
-  coal: 1288
-  hydro: 68
-  solar: 10.002
```

Comment:
when we implement the validation layer we need to make sure small values of production that is not in the capacity config is allowed.